Add SR2025 Kickstart event (#578)
srobo/website: _events/sr2025/kickstart.md

---
title: Kickstart
date: 2024-10-12 11:00:00 Europe/London
end_date: 2024-10-12 17:00:00 Europe/London
layout: event
type: kickstart
location: Southampton University
---

Kickstart is the event which kicks off a competition year. The structure of the
competition, the game rules will be announced and of course any questions you
have will be answered.

Throughout the day we'll have Blueshirts (our awesome volunteers!) around in
person, ready to answer any questions you may have and to help you through the
microgames. These microgames will help your team become familiar with the Kit
you've been provided and should kickstart some ideas for your robot's design.

If you are unable to attend Kickstart, please [let us know][teams-contact].

## Schedule

_To be announced._

## What to bring

> * Your robot kit (which will have been shipped to you beforehand)

Reviewer: There are many reasons this may not happen

```suggestion
* Your robot kit (which should have been shipped to you beforehand)
```

Author: Hrm, this is complicated by this being read both before and after we're telling people about this plan :thinking:

Author: https://github.com/srobo/website/pull/578/commits/82d8059ad20b15b0cbaa80de82dca15050370703

---
title: Kickstart
date: 2024-10-12 11:00:00 Europe/London
end_date: 2024-10-12 17:00:00 Europe/London
layout: event
type: kickstart
location: Southampton University
---

Kickstart is the event which kicks off a competition year. The structure of the
competition, the game rules will be announced and of course any questions you
have will be answered.

Throughout the day we'll have Blueshirts (our awesome volunteers!) around in
person, ready to answer any questions you may have and to help you through the
microgames. These microgames will help your team become familiar with the Kit
you've been provided and should kickstart some ideas for your robot's design.

> If you are unable to attend Kickstart, please [let us know][teams-contact].

Reviewer 2: Do we want to switch this to let us know if you want to attend?

Author: 8c4b33b348e32f5e04545993fb3da7341bb251fb

> If you are unable to attend Kickstart, please [let us know][teams-contact].

Reviewer 3: We probably want to mention something about streaming it

Author: 4ea43de106f420259bf24576ded9d86610ad71da

## Schedule

| Time  | Info |
|-------|------|
| 11:00 | Introductory presentation, where the game of SR2025 will be announced. |
| 12:30 | Lunch. |
| 13:30 | Microgames, a set of tasks to familiarise you with the kit. |
| 17:00 | Finish. |

## What to bring

> * Your robot kit (we will be shipping kits to teams ahead of Kickstart)

Reviewer 4: ```suggestion
* Your robot kit (we intend to ship kits to teams ahead of Kickstart)
```

Pending budget approval, etc.
